lib/govspeak.rb

```diff
@@ -414,6 +414,30 @@ def self.devolved_options
       lines.join
     end
 
+    extension("Details", /\$Details\s*$(.*?)\s*\$EndDetails/m) do |body|
+      lines = []
+
+      body.scan(/\$Heading\s*(.*?)\s*\$EndHeading\s*\$List\s*(.*?)\s*\$EndList/m) do |summary, list|
+
+        details_summary = Govspeak::Document.new(summary, allowed_elements: %w[details]).to_html.remove!("<p>").remove("</p>")
```

Comment:
Why do we need `allowed_elements` here?

Reply:
Also, maybe you can avoid `p` tags by passing in `<span class="govuk-details__summary-text">#{summary}</span>` rather than just `summary`? If not, can we make sure we only remove the first `p` and the last `/p`?

Reply:
Also also, `summary_html` is a better text ;)

Reply:
Looks like we don't know. I think this was left in before i worked out that you could add details to the `sanitize_config` method in html_santizer

Reply:
I can't seem to get rid of the <p> passing in the span but i think we want to remove all because we're passing it in to the span and don't want any p tags in there?